tests/psalm-baseline.xml

```diff
@@ -34,4 +34,9 @@
       <code>getId</code>
     </UndefinedInterfaceMethod>
   </file>
+  <file src="lib/Command/Export.php">
+    <InvalidArgument occurrences="1">
+      <code>$this</code>
+    </InvalidArgument>
+  </file>
```

Comment:
Why is that triggered?

Reply:
Just noticed the failure in the CI. Is it not possible to add the property to the stub?
Otherwise we should just move OC\Core\Command\Base to OCP, it makes sense to make it available for applications.

Reply:
Ok moving Command\Base to OCP was refused in https://github.com/nextcloud/server/pull/32332 so we have to wait on https://github.com/nextcloud/server/pull/32780
In the mean time add the property in the stub if possible, if not add it to the baseline.